Modules/_tkinter.c

```diff
@@ -357,11 +357,10 @@ static int Tkinter_busywaitinterval = 20;
 static void
 Sleep(int milli)
 {
-    /* XXX Too bad if you don't have select(). */
-    struct timeval t;
+    struct timespec t;
     t.tv_sec = milli/1000;
-    t.tv_usec = (milli%1000) * 1000;
-    select(0, (fd_set *)0, (fd_set *)0, (fd_set *)0, &t);
+    t.tv_nsec = (milli%1000) * 1000000;
+    nanosleep(&t, NULL);
```

Comment:
Please revert this change and write a second PR for this change.

Reply:
Luckily nanosleep() is supported in many unix OS, see my latest comment.
https://github.com/python/cpython/pull/28111#issuecomment-914642746

Do you suggest to do it in a new PR for changes of [semaphore.c](https://github.com/python/cpython/pull/28111/files#diff-cf698f2997f9f11b2851514321974536c9a161585e85034230a9783c4a7bc98e), also?